phys/module_sf_fogdes.F

```diff
@@ -111,9 +111,9 @@ SUBROUTINE sf_fogdes(&
 
 !------------------------------------------------------------------
 
-   IF     ((nlcat .eq. 20).or.(nlcat .eq. 21)) THEN ! includes lake category
+   IF     (mminlu .eq. 'MODIFIED_IGBP_MODIS_NOAH') THEN ! includes lake category
      land_use_type = 'MODIS'
-   ELSEIF ((nlcat .eq. 24).or.(nlcat .eq. 28)) THEN ! includes lake category
+   ELSEIF (mminlu .eq. 'USGS') THEN ! includes lake category
      land_use_type = 'USGS'
    ELSE
      PRINT *, 'Unknown landuse category (sf_fogdes.F): num_land_cat=',nlcat
```

Comment:
modify the print to output mminlu

Reply:
@WQadam Can you modify the print following your line of change to print mminlu, instead of nlcat? 
@dudhia I assume that is what you suggested?

Reply:
Sorry i have been busy the last couple of days. 
I printed it in my testing but forgot to put it in the github code. I have now made the error go into the rsl.error files instead of the error going into a random rsl.out file. Hopefully it looks okay and will never really be triggered.